sunpy/instr/goes.py

```diff
@@ -276,7 +380,7 @@ def goes_chianti_tem(longflux, shortflux, satellite=8,
     # GOES 6 long channel flux before 1983-Jun-28 must be corrected by a
     # factor of 4.43/5.32
     if date < datetime.datetime(1983, 06, 28) and satellite == 6:
-        longflux_corrected = longflux * (4.43/5.32)
+        longflux_corrected = longflux*(4.43/5.32)
```

Comment:
there already is a comment that it needs to be rescaled, maybe add a link, but I doubt there is docs :p